src/notebooks/R Mia Examples/Fetch-Analyses-metadata-for-a-Study.qmd

```diff
@@ -0,0 +1,98 @@
+---
+title: "Fetch analyses metadata for a study"
+author:
+  - name: Noah de Gunst
+    affiliation:
+      - id: mia
+        name: miaverse team
+categories: [R]
+execute: 
+  eval: true
+---
+
+::: {style="max-width:1200px"}
+![](./../_resources/mgnify_logo.png)
+
+# Fetch a study using MGnifyR; download the metadata for all of its analyses
+
+The [MGnify API](https://www.ebi.ac.uk/metagenomics/api/v1) returns data and relationships as JSON. [MGnifyR](https://github.com/beadyallen/MGnifyR) is a package to help you read MGnify data into your R analyses.
+
+You can find all of the other "API endpoints" using the [Browsable API interface in your web browser](https://www.ebi.ac.uk/metagenomics/api/v1).
+
+This is an interactive code notebook (a Jupyter Notebook). To run this code, click into each cell and press the ▶ button in the top toolbar, or press `shift+enter`.
+
+------------------------------------------------------------------------
+:::
+
+#### Setting the access code
+
+```{r}
+#| output: false
+
+source("./utils/variable_utils.r")
+
+mgnify_study_accession <- get_variable_from_link_or_input('MGYS', 'Study Accession', 
+                                                          'MGYS00005116')
+
+# You can also just directly set the accession variable in code, like this:
+# mgnify_study_accession <- "MGYS00005292"
+```
+
+#### Constructing a MgnifyClient object to access the database
+
+```{r}
+#| output: false
+# Importing the libraries
+library(vegan)
+library(ggplot2)
+library(mia)
+library(MGnifyR)
+
+mg <- MgnifyClient(usecache = TRUE, cache_dir = '/home/jovyan/.mgnify_cache')
```

Comment:
I expect that we shouldn't use local paths?

Reply:
I think this is the way they did it for the other examples - https://docs.mgnify.org/src/notebooks/R%20Examples/Comparative%20Metagenomics.html - so should we keep it as is?

Reply:
If it is possible to avoid local paths then we should do that.